pytorch_lightning/callbacks/gpu_stats_monitor.py

```diff
@@ -126,7 +126,7 @@ def setup(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule", stage: O
         if not trainer.logger:
             raise MisconfigurationException("Cannot use GPUStatsMonitor callback with Trainer that has no logger.")
 
-        if trainer._device_type != _AcceleratorType.GPU:
+        if trainer._accelerator_type != _AcceleratorType.GPU:
```

Comment:
(for a separate issue/PR) i know this is a rename, but why do we need this check at all? couldn't we use the strategy's root_device instead?

Reply:
Or strategy's `on_gpu` property. Could be a follow up.